src/v/raft/prevote_stm.cc

```diff
@@ -119,18 +122,18 @@ ss::future<bool> prevote_stm::prevote(bool leadership_transfer) {
       .with([this, leadership_transfer] {
           _config = _ptr->config();
           _config->for_each_voter(
-            [this](model::node_id id) { _replies.emplace(id, vmeta{}); });
+            [this](vnode id) { _replies.emplace(id, vmeta{}); });
 
           auto lstats = _ptr->_log.offsets();
           auto last_entry_term = _ptr->get_last_entry_term(lstats);
 
           _req = vote_request{
-            _ptr->_self,
-            _ptr->group(),
-            _ptr->term(),
-            lstats.dirty_offset,
-            last_entry_term,
-            leadership_transfer};
+            .node_id = _ptr->_self,
```

Comment:
`struct { .property = something }` is so much better that `struct { something }`, thanks for changing it!